labtool2.0/src/reducers/selectedInstanceReducer.js

```diff
@@ -37,6 +38,10 @@ const selectedInstanceReducer = (store = INITIAL_STATE, action) => {
       return { ...store, finalReview: action.value }
     case 'CI_MODIFY_ONE_SUCCESS':
       return { ...store, ...action.response, currentWeek: Number(action.response.currentWeek || store.currentWeek, 10) }
+    case 'SET_CR_VISIBLE':
+      return { ...store, currentCodeReview: store.currentCodeReview.concat(action.value) }
+    case 'HIDE_CR':
+      return { ...store, currentCodeReview: store.currentCodeReview.filter(cr => cr !== action.value) }
```

Comment:
I think this state should not be used to save checkbox selections as it would not be consistent with the values in database (i.e. navigating away from course modify page after setting code review visible without saving it would show the code review as visible even when it is not).

Reply:
You're right, I'll correct it